src/js/serial.js

```diff
@@ -52,8 +52,6 @@ var serial = {
                 });
 
                 self.onReceiveError.addListener(function watch_for_on_receive_errors(info) {
-                    console.error(info);
```

Comment:
Maybe is good to let here the message and not in the default section. In this way we can have always a message with the error, or include the `timeout` and `disconnected` message in the `default` part. I think it can be useful.

Reply:
Oh, the TODO note on `timeout` and `disconnected` are from cTn (baseflight). Should I just delete it so they are `default:` or have them still explicitly named?

Reply:
If they can happen, I will add them to the list that includes the default. If the cannot happen, I will remove this states.

Reply:
It's a situation where they don't happen for me, and they didn't happen for ctn. I can look for an explanation of the serial API. It's possible that the use of USB instead of real Serial ports causes this. I'll see what I can find.

I get "frame error" and "break" from my testing. In cli can save or exit. Can unplug.